### PR TITLE
Fix netty resource leak

### PR DESCRIPTION
**Pull request by e5l**
[`#KTOR-939`](https://youtrack.jetbrains.com/issue/KTOR-939)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -6,12 +6,14 @@ version=1.4.0
...
 kotlin.incremental.multiplatform=true
 kotlin.parallel.tasks.in.project=true
-org.gradle.parallel=true
 kotlin.native.ignoreIncorrectDependencies=true
 #kotlin.native.enableParallelExecutionCheck=false
 
 # gradle
+org.gradle.parallel=true
 org.gradle.daemon=true
+#org.gradle.configureondemand=true
```
Comment (cy6erGn0m) [inline gradle.properties:15]: irrelevant change

**Comment by cy6erGn0m** [comment]
Am I right that it's about unsarted servers? Would be nice to write a brief description here in PR for the future history

**Comment by e5l** [comment]
Sure. I linked the issue in the description and commit message

**Comment by cy6erGn0m** [comment]
The only side effect I can imagine is that concurrent start and stop could lead to undefined behaviour due to concurrent creating and disposing separate components.

**Comment by e5l** [comment]
I'm not sure if we should support concurrent `start` and `stop` call. Short lookup shows that also we don't support it in other engines

File: ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.server.netty
+
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import org.junit.*
```
Comment (cy6erGn0m) [inline ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt:9]: kotlin.test?

File: ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+class NettySpecificTest {
```
Comment (cy6erGn0m) [inline ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt:11]: Class name?

Comment (e5l) [reply ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt:11]: That's a good name for the test, but not for the suite I guess

File: ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt (hunk 2 above)
Comment (e5l) [reply ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt:9]: +

**Review by cy6erGn0m — APPROVED** [review]

**Pull request merged**